docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -99,6 +100,19 @@ Restore succeeded.
 ```
 
   </TabItem>
+
+  <TabItem value="java" label="Java (Spring Boot)">
+
+For Java, we will use the [Spring Initializr](https://start.spring.io/) to generate the example source code:
+
+1. Navigate to [start.spring.io](https://start.spring.io/) 
+2. Select **11** under **Java**
+2. Click on "Add" under "Dependencies"
+3. Select "Spring Web"
+4. Click "Generate" to generate and download the source code
```

Comment:
Just like the instructions for Node.JS or .NET (which use the CLI), maybe you could mention the ability to use the [Spring Boot CLI](https://docs.spring.io/spring-boot/docs/current/reference/html/cli.html) to bootstrap a new project, like so. People familiar with Spring Boot might be already using the CLI as well, I think.

```shell
❯ spring init --dependencies=web --build=maven --language=java --java-version=11 my-springboot-project
Using service at https://start.spring.io
Project extracted to '/tmp/spring-boot-using-cli/my-springboot-project'
```

Otherwise, it is also possible to call the Spring Initializr API to generate a new project, like so:

```shell
❯ curl https://start.spring.io/starter.zip  \
  -d baseDir=my-springboot-project \
  -d javaVersion=11 \
  -d dependencies=web \
  -d language=java \
  -d type=maven-project --output my-springboot-project.zip && \
unzip -q my-springboot-project.zip && \
rm -rf my-springboot-project.zip
```

It is much faster than clicking through the Initializr web UI, I think.

Reply:
Good catch! Let's give both options as being available